### DISCUSSION TITLE
Is it possible to upgrade a cask without uninstalling it first?

### DISCUSSION THREAD

**Discussion by J-M0** [Homebrew/discussions#581] [Casks]
This is a small annoyance, but since casks are uninstalled before their updates are installed, that means their icons are removed from my Launchpad and I have to reorganize them after every update. This doesn't happen in the non-cask workflow of just installing an update over the current installation whether that's by moving an `.app` or installing a `.pkg`.

**Comment by SMillerDev** [comment]
AFAIK no, this is not possible.

CC @Homebrew/cask

**Comment by vitorgalvao** [comment]
No, it is not possible. https://github.com/Homebrew/homebrew-cask/issues/89497.
